metrics/prometheus.go

```diff
@@ -1595,6 +1595,82 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.ResctrlMetrics) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:        "container_mem_bandwidth_bytes",
```

Comment:
use container_memory instead of container_mem to be consistent with other memory metrics.

Reply:
also, cumulative metrics should have the suffix `_total`